roles/rpm_ostree_livefs/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# vim: set ft=ansible:
+#
+- name: Enable livefs
+  command: rpm-ostree ex livefs
+  register: ros_lfs
+
+# Regex to find all the sha256 checksums in the output
+- name: Get commits
+  set_fact:
+    commits: "{{ ros_lfs.stdout|regex_findall('[A-Fa-f0-9]{64}') }}"
```

Comment:
This data is also in `rpm-ostree status --json` (key `live-replaced` etc.); scraping it is going to be fragile.  There's code in some of the other tasks that parse that right?

Reply:
I'm using the checksum from the output to verify the live-replaced in the rpm-ostree status output.  Is there a way to generate the checksum directly so I can use it to verify the livefs output and rpm-ostree status output?

Reply:
I think it would be reasonable to use `rpm-ostree status --json` and then just pull the commit data we want from there:

```
- name: Enable livefs
  command: rpm-ostree ex livefs
  register: ros_lfs

- name: Get status
  command: rpm-ostree status --json
  register: ros

- name: Convert to JSON
  set_fact:
    ros_json: "{{ ros.stdout | from_json }}"

- name: Query for commits
  set_fact:
    rol_base_commit: "{{ ros_json['deployments'][0]['base-checksum'] }}"
    rol_livefs_commit: "{{ ros_json['deployments'][0]['checksum'] }}"
```

This might be a little fragile because we are assuming that deployment 0 is the pending deployment that was made live, but we should have enough confidence in the state of the deployments when using this role.

Reply:
After talking to @mike-nguyen on IRC, I'm going to retract my suggestion here.  

The intent is to compare the commit IDs from the output of `rpm-ostree ex livefs` to the output from `rpm-ostree status`, so my suggestion doesn't quite fit.